tests/dragonfly/seeder/script-genlib.lua

```diff
@@ -114,7 +115,7 @@ function LG_funcs.add_hash(key, keys)
     local blobs
     local is_huge = keys[key]
     if is_huge then
-        blobs  = dragonfly.randstr(LG_funcs.huge_value_size, LG_funcs.csize / 2)
+        blobs  = dragonfly.randstr(LG_funcs.huge_value_size, LG_funcs.huge_value_csize)
```

Comment:
can you explain how this is working if line 129 does 
for i = 2, LG_funcs.csize, 2 do